### PR TITLE
Docs: Update Linux installation instructions

### PR DESCRIPTION
**Pull request by dolang**
**WIP** &mdash; please review and criticise.

I'll focus on content in this one. Organisation/Layout later. Will probably need help with Bodhi Linux and Gentoo.

### Linux
- Prebuilt:
  - [x] Ubuntu PPA
  - [x] Debian (tested the showcase with kivy-daily; Stretch 9.4 only. Is that enough?)
  - [x] Linux Mint
  - ~~Bodhi Linux (?)~~
  - [x] OpenSUSE
  - [x] Gentoo
  - [ ] Fedora (?)
- Virtual Environment:
  - [ ] Ubuntu
  - [ ] Debian
  - [ ] Linux Mint
  - [ ] Bodhi Linux (?)
  - [ ] OpenSUSE
  - [ ] Gentoo
  - [ ] Fedora (?)
- [ ] PyGame (?)
- [ ] Start from the Command Line (examples)
- [ ] Device permissions

### CODE REVIEW

**Comment by inclement** [comment]
Looks good to me so far.

About PyGame, I think it's fine to just not document that.

**Comment by inclement** [comment]
Yep, pygame support is unlikely to actually stop working (at least deliberately), but it's basically completely deprecated as far as dev support is concerned.

**Comment by bionoid** [comment]
FYI @dolang I have some old work on linux install docs squashed in this commit:

https://github.com/bionoid/kivy/commit/5795ce80388d7ff4f257c7c4745edfa4ef1877d7

Some things are maybe worth copying or using as a foundation, such as RHEL7+Fedora build instructions, the sdl2_from_source.txt, etc.

**Comment by dolang** [comment]
@bionoid ah, very nice. I'll certainly look through that and see what I can adapt.

File: doc/sources/installation/installation-linux.rst [hunk 1]
```diff
@@ -3,120 +3,167 @@
...
 
-#. Add the GPG key to your apt keyring by executing
+Debian
+------
 
-    as user:
+The situation here is similar to Ubuntu's. Both official packages are still
+based on version 1.9.1, and ``python3-kivy`` is currently broken. If you decide
+to install ``python-kivy`` you may want to add the optional ``python-pygments``
+and ``python-docutils`` as well.
```
Comment (Zen-CODE) [inline doc/sources/installation/installation-linux.rst]: and ``python-docutils`` packages as well.

**Comment by AndreMiras** [comment]
Hi @dolang I don't know how this got forgotten, the changes were pretty good and now master moved and it's conflicting :cry: 
Would you mind solving the conflict so I we can merge it?

**Comment by matham** [comment]
I'm closing for now. Please re-open once it can be merged and if you think it's ready (I see you still have the WIP tag).

**Pull request closed without merging**